lib/authentication/auth_web.js

```diff
@@ -19,12 +21,13 @@ var URLUtil = require('./../../lib/url_util');
  * @returns {Object}
  * @constructor
  */
-function auth_web(host, webbrowser, httpclient)
+function auth_web(host, webbrowser, httpclient, browserActionTimeout)
```

Comment:
we can get it from global config, not as parameter

Reply:
It is connection config parameter, doesn't exist in global config